ImportError: No module named 'sklearn.cross_validation'の対処
===================================================================

sklearnのバージョンの不一致が原因。

sklearn 0.20からcross_validationモジュールは消滅し、代わりにmodel_selectionモジュールを使うことになった。

（model_selectionモジュール自体は0.19とかでも存在する）

なお、sklearnのバージョンは次のようなコマンドで確認できる。

ただし、環境によっては微妙にコマンドが違う可能性もある（pip3だったりcondaだったり。

常識的なことだが、念のために書いておく）。

# 古い書き方
from sklearn.cross_validation import cross_val_score

# 新しい書き方
from sklearn.model_selection import cross_val_score